libbpfgo.c

```diff
@@ -6,8 +6,8 @@ extern void perfLostCallback(void *ctx, int cpu, __u64 cnt);
 extern int ringbufferCallback(void *ctx, void *data, size_t size);
 
 int libbpf_print_fn(enum libbpf_print_level level, // libbpf print level
-                    const char *format,            // format used for the msg
-                    va_list args)                  // args used by format
+                    const char *format, // format used for the msg
```

Comment:
I do prefer the old indentation, it resembles Golang formatting. Which clang-format option should be returned?

Reply:
I think it's `AlignTrailingComments`.